docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -136,30 +103,40 @@ Pick one of the following plugins:
 
   module.exports = {plugins};
   ```
+
+If you are using Create React App, there is no need for a `.babelrc.js` file and the configuration is easy by using [craco](https://github.com/sharegate/craco):
+
+  `yarn add -D @craco/craco babel-plugin-transform-imports`
   
-If you are using Create React App, you will need to use a couple of projects that let you use `.babelrc` configuration, without ejecting. 
-  
-  `yarn add -D react-app-rewired customize-cra`
-  
-  Create a `config-overrides.js` file in the root directory:
+  Create a `craco-config.js` file in the root directory:
 
   ```js
-  /* config-overrides.js */
-  const { useBabelRc, override } = require('customize-cra')
-
-  module.exports = override(
-    useBabelRc()
-  );  
+/* eslint-disable no-template-curly-in-string */
```

Comment:
Nit: Remove the eslint-disable here. It depends on the ruleset of the app. Aside: This might be a good indicator that you shouldn't support this syntax to avoid confusion.